MAUI/NavigationDrawer/Multi-Drawer.md

```diff
@@ -0,0 +1,146 @@
+---
+layout: post
+title: Multi Drawer in .NET MAUI Navigation Drawer | Syncfusion®
+description: The navigation drawer allows users to open drawers on multiple sides with different toggle methods, offering customizable layouts and smooth transitions.
+platform: MAUI
+control: SfNavigationDrawer
+documentation: UG
+---
+
+# Multi Drawer in MAUI Navigation Drawer (SfNavigationDrawer)
+
+The navigation drawer allows users to open the drawer on multiple sides with different toggle methods. The [DrawerSettings](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.NavigationDrawer.SfNavigationDrawer.html#Syncfusion_Maui_NavigationDrawer_SfNavigationDrawer_DrawerSettings) class and its properties need to be used when users need to provide multiple drawers. The multiple drawers can be implemented using the following drawer settings.
+
+* DrawerSettings
+* SecondaryDrawerSettings
+
+N> Users can open only one drawer at a time.
+
+![Multi Drawer](Images/multi-drawer/multi-drawer.gif)
+
+### DrawerSettings
+
+Implement the primary drawer using the drawer settings class. The following code sample demonstrates how to set the drawer settings’s properties inside the [DrawerSettings](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.NavigationDrawer.SfNavigationDrawer.html#Syncfusion_Maui_NavigationDrawer_SfNavigationDrawer_DrawerSettings) class.
```

Comment:
Implement the primary drawer using the DrawerSettings property in SfNavigationDrawer. The following code sample demonstrates how to customize the primary drawer.